network/src/protocols/discovery/test.rs

```diff
@@ -187,7 +185,17 @@ fn inbound() {
         delivered_rx.await.unwrap();
 
         // Connectivity manager receives address of other peer.
-        expect_address_update(&mut conn_mgr_reqs_rx, other_peer_id, &other_addrs[..]).await;
+        expect_address_update(
+            &mut conn_mgr_reqs_rx,
+            [
+                (other_peer_id, other_addrs),
+                (self_peer_id, self_addrs.clone()),
+            ]
+            .iter()
+            .cloned()
```

Comment:
does `.into_iter()` instead of `.iter().cloned()` work here?

Reply:
> value of type `std::collections::HashMap<move_core_types::account_address::AccountAddress, std::vec::Vec<libra_network_address::NetworkAddress>>` cannot be built from `std::iter::Iterator<Item=&(move_core_types::account_address::AccountAddress, std::vec::Vec<libra_network_address::NetworkAddress>)>`

Reply:
😭